README.md

```diff
@@ -311,6 +315,34 @@ MFA to the right AWS account using the following command
 
     tx-mfa
 
+## Production Deployment
+
+### Prerequisites
+
+The production pipeline terraform stack must be deployed
+
+    make deploy-deployment-pipelines PROFILE=tools ENVIRONMENT=dev
```

Comment:
Can I check this is meant to be the `dev` environment and not `mgmt` ?

Reply:
Yes I was following Ed 
![image](https://user-images.githubusercontent.com/62281988/158412294-7f4ab214-d0cb-4cc1-a402-fe6d0e0fcd64.png)
![image](https://user-images.githubusercontent.com/62281988/158412341-01742e40-18cc-4fe4-83e0-577a87af936e.png)